views/guide.lavender

```diff
@@ -317,12 +322,12 @@ block content
             | and
             code.language-markup false
             | )
-            | are keywords in Lavender, they can be assinged to variables or used in 
-            | conditionals directly (but if you actually do that then
+            | are keywords in Lavender. They can be assinged to variables, or used in 
```

Comment:
Assinged! He was assinged by the flames of glory... Should be assigned, oops by probably me